src/script/media/MediaDevicesHandler.ts

```diff
@@ -154,7 +155,12 @@ export class MediaDevicesHandler {
       videoinput: this.availableDevices.videoinput().length,
     };
 
-    this.initializeMediaDevices();
+    // The device list must be queried once to obtain the device IDs. This way, the frontend knows whether cameras
+    // and microphones exist and can request them specifically during a call. Additionally, the app needs to know
+    // the device IDs; otherwise, we will receive a Media-Query-Constraint error when querying the devices, as we
+    // explicitly query by the device ID.
+    // The false parameter ensures that we only load the list temporarily.
+    this.initializeMediaDevices(false);
```

Comment:
To maintain the old behavior and because we need the device IDs, I will temporarily initialize the device list at startup.

Reply:
False' essentially means that no camera will be started. Without a video track, Firefox doesn't have permissions for the labels. Therefore, the device list will not be saved as final!